plugins/formal-verification/formver.plugin/src/org/jetbrains/kotlin/formver/UniqueDeclarationChecker.kt

```diff
@@ -20,9 +22,10 @@ class UniqueDeclarationChecker(private val session: FirSession, private val conf
         if (!config.checkUniqueness) return
         val errorCollector = ErrorCollector()
         try {
-            val uniqueCheckerContext = UniqueChecker(session, config, errorCollector)
-            declaration.accept(UniquenessCheckExceptionWrapper, uniqueCheckerContext)
+            val cfaChecker = UniqueCFA(UniqueCheckerData(session, config, errorCollector))
+            declaration.controlFlowGraphReference?.controlFlowGraph?.let { cfaChecker.analyze(it, reporter, context) }
         } catch (e: Exception) {
+            errorCollector.addErrorInfo("... while checking uniqueness level for ${declaration.source.text}")
```

Comment:
We use this kind of info when producing internal errors.  I would actually not use `ErrorCollector` here at all; you already have the `reporter`.